packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -32,48 +33,55 @@ function SharePostContent({
     <>
       <p className="mt-6 whitespace-pre-line typo-title3">{post.title}</p>
       <div className="flex flex-col mt-8 rounded-16 border border-theme-divider-tertiary hover:border-theme-divider-secondary">
-        <a
+        <Link
           href={
             post.sharedPost.source.id === 'unknown'
               ? post.sharedPost.permalink
+              : `${post.sharedPost.commentsPermalink}?squad=${post.source.name}`
+          }
+          as={
+            post.sharedPost.source.id === 'unknown'
+              ? undefined
               : post.sharedPost.commentsPermalink
           }
-          title="Go to post"
-          target="_blank"
-          rel="noopener"
-          className="flex flex-col-reverse laptop:flex-row p-4 max-w-full"
-          onClick={onReadArticle}
         >
-          <div className="flex flex-col flex-1">
-            <h2 className="flex flex-wrap mt-4 laptop:mt-0 mb-4 font-bold typo-body">
-              {post.sharedPost.title}
-            </h2>
-            <PostSourceInfo
-              date={
-                post.sharedPost.readTime
-                  ? `${post.sharedPost.readTime}m read time`
-                  : undefined
-              }
-              source={post.sharedPost.source}
-              size="small"
-            />
-            <ReadArticleButton
-              className="mt-5 btn-secondary w-fit"
-              href={post.sharedPost.permalink}
-              openNewTab={openNewTab}
-              onClick={onReadArticle}
-            />
-          </div>
-          <div className="block overflow-hidden ml-2 w-70 rounded-2xl cursor-pointer h-fit">
-            <LazyImage
-              imgSrc={post.sharedPost.image}
-              imgAlt="Post cover image"
-              ratio="52%"
-              eager
-              fallbackSrc={cloudinary.post.imageCoverPlaceholder}
-            />
-          </div>
-        </a>
+          {/* eslint-disable-next-line react/jsx-no-comment-textnodes, jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
```

Comment:
Needed to add this since `a` without `href` was triggering eslint rules and our nextjs version does not support `onClick` on `Link`.